website/docs/r/servicebus_subscription.html.markdown

```diff
@@ -92,7 +92,7 @@ The following arguments are supported:
 
 ### TimeSpan Format
 
-Some arguments for this resource are required in the TimeSpan format which is used to represent a length of time. The supported format is documented [here](https://msdn.microsoft.com/en-us/library/se73z7b9(v=vs.110).aspx#Anchor_2)
+Some arguments for this resource are required in the TimeSpan format which is used to represent a length of time. The supported format is documented [here](https://docs.microsoft.com/en-us/cli/azure/servicebus/topic/subscription?view=azure-cli-latest#az-servicebus-topic-subscription-update). Example value: "P3DT0H0M0.0S" (for 3days)
```

Comment:
When adding a link to the docs site, can we remove the locale from the URL to allow the site to auto-detect the users locale?

```suggestion
Some arguments for this resource are required in the TimeSpan format which is used to represent a length of time. The supported format is documented [here](https://docs.microsoft.com/cli/azure/servicebus/topic/subscription?view=azure-cli-latest#az-servicebus-topic-subscription-update). Example value: "P3DT0H0M0.0S" (for 3days)
```